docs/types.rst

```diff
@@ -171,6 +171,19 @@ Fixed Point Numbers
 
 **COMING SOON...**
 
+.. index:: address, literal;address
+
+.. _address_literals:
+
+Address Literals
+----------------
+
+Hexadecimal literals that pass the address checksum test, for example
+``0xdCad3a6d3569DF655070DEd06cb7A1b2Ccd1D3AF`` are of ``address`` type.
+Hexadecimal literals that are 20 bytes long (differ from that in at
```

Comment:
For clarity, should we just say that "Hexadecimal literals that are 39 to 41 digits long, but do not..."?